agent/command/expansion_write.go

```diff
@@ -7,11 +7,21 @@ import (
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/agent/internal"
 	"github.com/evergreen-ci/evergreen/agent/internal/client"
+	"github.com/evergreen-ci/utility"
 	"github.com/mitchellh/mapstructure"
 	"github.com/pkg/errors"
 	yaml "gopkg.in/yaml.v3"
 )
 
+var (
+	expansionsToRedact = []string{
+		evergreen.GlobalGitHubTokenExpansion,
+		AwsAccessKeyId,
```

Comment:
nit: can we call these AWSAccessKeyId and etc? It's convention to capitalize all the letters in an acronym.